data/org.freedesktop.portal.Device.xml

```diff
@@ -58,6 +58,66 @@
       <arg type="a{sv}" name="options" direction="in"/>
       <arg type="o" name="handle" direction="out"/>
     </method>
+
+    <!--
+        OpenPipeWireRemote:
+        @options: Vardict with optional further information
+        @fd: File descriptor of an open PipeWire remote.
+
+        Open a file descriptor to the PipeWire remote where the microphone,
+        speakers, and camera nodes are available. The file descriptor should be
+        used to create a <classname>pw_core</classname> object, by using
+        <function>pw_context_connect_fd</function>.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>devices as</term>
+            <listitem><para>
+              The devices to access. Supported devices for this call are 'camera',
+              'microphone', and 'speakers'.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+
+        This method will only succeed if the application already has permission
+        to access any one of the devices requested.
```

Comment:
"any one" or "all"?

Reply:
"any one" is the accurate wording

Reply:
It might be useful to change "one" to a different word to avoid ambiguity of English language. maybe "single"?